moveit_core/robot_state/include/moveit/robot_state/robot_state.h

```diff
@@ -1749,13 +1741,21 @@ class RobotState
   void initTransforms();
   void copyFrom(const RobotState& other);
 
+  /**
+   * \brief Transform pose from the robot model's base frame to the reference frame of the IK solver
+   * @param pose - the input to change
+   * @param solver - a kin solver whose base frame is important to us
+   * @return true if no error
+   */
+  inline bool setToIKSolverFrame(Eigen::Isometry3d& pose, const kinematics::KinematicsBaseConstPtr& solver);
+
   /**
    * \brief Transform pose from the robot model's base frame to the reference frame of the IK solver
    * @param pose - the input to change
    * @param ik_frame - the name of frame of reference of base of ik solver
    * @return true if no error
    */
-  bool setToIKSolverFrame(Eigen::Isometry3d& pose, const std::string& ik_frame);
+  inline bool setToIKSolverFrame(Eigen::Isometry3d& pose, const std::string& ik_frame);
```

Comment:
This one shouldn't be inline. It's a longer function. However, the other one is just forwarding to this one here.
```suggestion
  bool setToIKSolverFrame(Eigen::Isometry3d& pose, const std::string& ik_frame);
```